scripts/dev/sup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+if [ "$KAZOO_CONFIG" = "" ]
+then
+	export KAZOO_CONFIG=$PWD/config.ini
+fi
+
+if [ "$KAZOO_NODE" = "" ]
+then
+	KAZOO_NODE=kazoo@localhost
+fi
+
+IFS='@' read -r -a NH <<< "$KAZOO_NODE"
+NAME="${NH[0]}"
+HOST="${NH[1]}"
```

Comment:
To supply sup with NAME and HOST deduced from KAZOO_NODE, to make it talk to this node.